doc/architectural_decisions/001-repo-structure.md

```diff
@@ -15,25 +15,25 @@ Tom & Kathryn
 
 ## Decision
 
-We will create a `core` repository, and publish it on PyPI.
+We will create a `core` repository, called `ibex_bluesky_core`, and publish it on PyPI.
 
 This repository will provide core building blocks, including plan stubs,
 devices, and utilities which are generic and expected to be useful across
 different science groups.
 
-Beamline or technique specific repositories will then depend on the `core`
-repository via PyPI.
+~~Beamline or technique specific repositories will then depend on the `core` repository via PyPI.~~
+Superseded by [ADR 6](006-where-to-put-code.md).
 
-The core repository will not depend on `genie_python`, so that other groups
-at RAL can use this repository. The genie python *distribution* may in future
-depend on this repository.
+The core repository will not depend on [`genie_python`](https://github.com/isiscomputinggroup/genie), so that other
+groups at RAL can use this repository. The [uktena](https://github.com/isiscomputinggroup/uktena) python *distribution* 
+depends on this repository.
```

Comment:
more installs than depends on? I don't think much would break if we actually removed it other than the get_run_engine() in the gui startup?